cmd/cli/main.go

```diff
@@ -168,6 +168,33 @@ func UpdateAction(c *cli.Context) error {
 	return nil
 }
 
+// TerminateAction executes the CLI terminate command
+func TerminateAction(c *cli.Context) error {
+	terminate := operations.TerminateJob{}
+
+	jobNameBase := c.String("job-name-base")
+	if len(jobNameBase) != 0 {
+		terminate.JobNameBase = jobNameBase
+	} else {
+		return cli.NewExitError("unspecified flag 'job-name-base'", -1)
+	}
+
+	mode := c.String("mode")
+	if len(mode) > 0 && mode != "cancel" && mode != "stop" {
+		return cli.NewExitError("unknown value for 'mode', only 'cancel' and 'stop' are supported", -1)
+	}
```

Comment:
Nitpicking again but put `terminate.Mode = mode` directly on 186 directly after the closing bracket on 185 and keep 1 empty line between 186 and the current 188 where you call `operator.Terminate`